core/src/main/java/org/neo4j/graphalgo/core/GraphLoader.java

```diff
@@ -129,23 +128,41 @@ public GraphLoader withAnyRelationshipType() {
 
     /**
      * Instructs the loader to load weights by reading the given property.
-     * If the property is not found, no weights are loaded.
+     * If the property is not found, the propertyDefaultValue is used instead.
      *
-     * @param property May not be null; to remove a weight property, use {@link #withoutWeights()} instead.
+     * @param property May not be null; to remove a weight property, use {@link #withoutWeights(double)} or
+     * {@link #withoutWeights()} instead.
+     * @param propertyDefaultValue the default value to use if property is not set
      * @return itself to enable fluent interface
      */
-    public GraphLoader withWeightsFromProperty(String property) {
+    public GraphLoader withWeightsFromProperty(String property, double propertyDefaultValue) {
         this.property = Objects.requireNonNull(property);
+        this.propertyDefaultValue = propertyDefaultValue;
         return this;
     }
 
     /**
      * Instructs the loader to not load any weights.
+     * The graph is initialized with the propertyDefaultValue instead.
+     *
+     * @param propertyDefaultValue the default value.
+     * @return itself to enable fluent interface
+     */
+    public GraphLoader withoutWeights(double propertyDefaultValue) {
```

Comment:
Oh perhaps I mixed it up, I thought this was the method to provide a default weight if there is no property on a relationship. That one seems to be missing then?